backend/src/main/java/gov/cdc/usds/simplereport/api/pxp/PatientExperienceController.java

```diff
@@ -115,5 +116,6 @@ public void patientLinkSubmit(@RequestBody PxpApiWrapper<AoEQuestions> body)
         data.getPriorTestResult() == null ? null : TestResult.valueOf(data.getPriorTestResult()),
         data.getSymptomOnset(),
         data.getNoSymptoms());
+    pls.expireMyPatientLink();
```

Comment:
Should they only get one shot at answering the questions? Or should the questions be available until the test is completed?

Reply:
This is maybe a design/product question, but I agree with Ben here. Should `updateMyTimeOfTestQuestions` just fail if the test has already been completed? That would let the patient experience skip messing with the expiration date.

Reply:
My understanding was that yes, the patient should only get one chance at answering the questions, and once they're answered their link shouldn't work again. Will check with design & come back to this thread

Reply:
Per Alicia, this is fine as-is